grunttasks/l10n-extract.js

```diff
@@ -22,10 +22,13 @@ module.exports = function (grunt) {
 
     var clientWalker = extract({
       'input-dir': path.join(__dirname, '..', '.es5', 'scripts'),
```

Comment:
With the newest jsxgettext, do we need to run this on the babelified scripts? Can we skip the babel step?